x-pack/plugins/apm/public/components/app/settings/agent_explorer/index.tsx

```diff
@@ -120,6 +125,15 @@ export function AgentExplorer() {
       <EuiSpacer />
       <EuiFlexItem>
         <EuiFlexGroup justifyContent="flexEnd">
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup gutterSize="xs">
```

Comment:
Why do you need an `EuiFlexGroup` in this case?

Reply:
You are right, I thought I was fixing the size of the selector by using that group but it's not the case 😅